build.gradle

```diff
@@ -2,10 +2,10 @@ apply plugin: 'java'
 apply plugin: 'war'
 
 buildscript {
-  repositories { jcenter() }
-  dependencies {
-    classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.1'
-  }
+    repositories { jcenter() }
+    dependencies {
+        classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.4'
```

Comment:
Yes. For my own test script to work, I need shadow to preserve mtime for .class files in the dependencies.
1.2.1 does not use the original mtime.